### ISSUE TITLE
Notification overlaps Dropdown-Menu

### ISSUE THREAD

**Issue by MaxMatti** [immich-app/immich#13196]
### The bug

![image](https://github.com/user-attachments/assets/07c2026f-9a9f-4602-b327-0b34977e774b)
Not sure if this counts as issue or feature requests...

Perhaps the dropdown can "push the notification down" i.e. to position the notifications the height of the dropdown further down when the dropdown is shown?

### The OS that Immich Server is running on

Ubuntu 24.04

### Version of Immich Server

v1.115.0

### Version of Immich Mobile App

-

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

1. Click on "people" in sidebar
2. Click on a face
3. Select a bunch of pictures
4. Add them to an album
5. Open the options menu by clicking on the three dots in the top right

### Additional information

Also when archiving pictures from that view, they are not shown until the view is reloaded. If other pictures are present that have previously been archived they are still shown but not the pictures that were just archived which is a bit weird.

**Comment by abhishek-singh0710** [comment]
I would like to work on this. Could this be assigned to me?
Thank you.

**Comment by Pranay-Pandey** [comment]
I don't think we can push down the notifications easily. The notifications are a a completely different component from the menu items. I think we can place the notifications at the bottom. So instead of notifications starting from the top to the bottom of the page, they start from the bottom and go to the top ?

**Comment by shadowlocked** [comment]
It would be better for me to be able to actually disable the notifications. In a high-volume session, these confirmations add a ton of time to the workflow. If you don't want to expose that functionality to the casual user, perhaps you could make it a flag in a text-based config that could be opened and altered.
